Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-05
Initial Package Version: 7.07.1rc2
Origin: BLFS Book
Description: Add missing png header.
--- espgs-7.07.1rc2/src/gdevpng.c.orig	2003-09-14 19:06:37.000000000 +0000
+++ espgs-7.07.1rc2/src/gdevpng.c	2003-09-14 19:06:48.000000000 +0000
@@ -34,6 +34,7 @@
 /* lpd (L. Peter Deutsch) 1996-4-7: Modified for libpng 0.88. */
 /* Original version by Russell Lang 1995-07-04 */
 
+#include <png.h>
 #include "gdevprn.h"
 #include "gdevmem.h"
 #include "gdevpccm.h"
